contracts/staker/BorrowStaker.sol

```diff
@@ -174,7 +174,7 @@ abstract contract BorrowStaker is BorrowStakerStorage, ERC20PermitUpgradeable {
     /// @notice Returns the exact amount that will be received if calling `claim_rewards(from)` for a specific reward token
     /// @param from Address to claim for
     /// @param _rewardToken Token to get rewards for
-    function claimableRewards(address from, IERC20 _rewardToken) external view returns (uint256) {
+    function claimableRewards(address from, IERC20 _rewardToken) external virtual returns (uint256) {
```

Comment:
Ref - https://arbiscan.io/address/0x63F00F688086F0109d586501E783e33f2C950e78#code (ctrl+f on `earned(`). If off chain and we let the interface as view it should be good. Also note that only the one that can not be view functions are not flagged because it is overriden if only view calls